eng/targets/Helix.Common.props

```diff
@@ -22,9 +22,9 @@
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' != 'true'">
     <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Windows.11.Amd64.ClientPre.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="OSX" />
+    <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="OSX" />
   </ItemGroup>
-  
+
   <!-- x64 Quarantined-only (quarantined-pr.yml and quarantined-tests.yml) test queues -->
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(_UseHelixOpenQueues)' == 'true' AND '$(RunQuarantinedTests)' == 'true'">
     <HelixAvailableTargetQueue Include="$(HelixQueueFedora34)" Platform="Linux" />
```

Comment:
I think this was the queue w/ `libmsquic` installed, we should keep it in both (at least for now) since it's the only place where we can run HTTP/3 tests on Linux (CC @JamesNK @Tratcher @adityamandaleeka)

Reply:
Do you mean all HTTP/3 tests are quarantined❔ My question was only about the duplication between `aspnetcore-quarantined-tests` (a bit lower down) and `aspnetcore-quarantined-pr`.

Reply:
No, most are no longer quarantined. As long as the Fedora34 queue is used in at least one non-quarantine job, it should be fine to remove it from either `quar-tests` or `quar-pr`

Reply:
:ok: I'll remove this block in the next iteration. That'll realign `aspnetcore-ci` and `aspnetcore-quarantined-pr` queues. `aspnetcore-helix-matrix` and `aspnetcore-quarantined-tests` were already aligned and _both_ include the Fedora 34 queue.